### PR TITLE
wgsl: Fix typo in `modf` description

### PR DESCRIPTION
**Pull request by ben-clayton**

### CODE REVIEW

**Review by alan-baker — CHANGES_REQUESTED** [review]
This makes sense, but the f16 version doesn't. Mind changing that one too?

File: wgsl/index.bs [hunk 1]
```diff
@@ -12439,10 +12439,10 @@ Note: The result is not mathematically meaningful if `e` &le; 0.
     * 1023 for AbstractFloat, when AbstractFloat is [[!IEEE-754|IEEE-754]] binary64
 
     If `x` is zero or a finite normal value for its type, then:
-  
-    <blockquote>                
+
+    <blockquote>
     x = ldexp(frexp(x).sig, frexp(x).exp)
```
Comment (munrocket) [inline wgsl/index.bs:12444]: .sig -> .fract

**Review by munrocket — CHANGES_REQUESTED** [review]

File: wgsl/index.bs (hunk 1 above)
Comment (dneto0) [reply wgsl/index.bs:12444]: We can land that separately.  :-)

**Review by dneto0 — APPROVED** [review]

File: wgsl/index.bs (hunk 1 above)
Comment (dneto0) [reply wgsl/index.bs:12444]: thank you for #3624

**Comment by ben-clayton** [comment]
> This makes sense, but the f16 version doesn't. Mind changing that one too?

Done

**Review by ben-clayton — COMMENTED** [review]
Abandoning for #3625

**Pull request closed without merging**